adaptor/adaptor.go

```diff
@@ -86,6 +96,11 @@ func (vm adaptor[BP]) BuildBlock(ctx context.Context) (snowman.Block, error) {
 	return vm.newBlock(vm.ChainVM.BuildBlock(ctx))
 }
 
+// BuildBlockWithContext calls BuildBlockWithContext(ctx, blockCtx) on the [ChainVM] that created b.
```

Comment:
```suggestion
```

This is internal so isn't strictly required. It's also incorrect since there is no `b` argument nor variable.